syslog/bin/import_logs.py

```diff
@@ -938,7 +938,7 @@ def _parse_args(self, option_parser, argv = None):
 
         if self.options.output:
             sys.stdout = sys.stderr = open(self.options.output, 'a+', 0)
-
+            self.options.output.close()
```

Comment:
I think this needs to stay open for the lifetime of the script execution for it to append logs as expected.

Reply:
I moved the file closures to after main() method runs and before we exit 0.